mesonpy/__init__.py

```diff
@@ -731,11 +796,23 @@ def _wheel_builder(self) -> _WheelBuilder:
             self._copy_files,
         )
 
+    def build_commands(self, install_dir: Optional[pathlib.Path] = None) -> Collection[Sequence[str]]:
+        return (
+            ('meson', 'compile', *self._meson_args['compile'],),
```

Comment:
This is now broken for me again.

On import, this is invoking just plain ``meson compile`` without a builddir, which is wrong.